orttraining/orttraining/test/graph/bert_toy_fetches.h

```diff
@@ -8,8 +8,9 @@
+// Avoid this arrary being initialized on stack.
```

Comment:
--> array

Reply:
@PeixuanZuo Can you add some comment for this workaround, something similar to our Teams chat?

First this line causes 
> fatal error: instantiating fold expression with 16383 arguments exceeded expression nesting limit of 256
in some version of clang (ROCm hip-clang is using)

> By eliminating the trailing zeros, the errors will be fixed, so we switch to explicitly set the array size and relay on the fact that this global variable has static storage duration then all those remaining elements will be zero initialized.